docs/tasks/debug-application-cluster/run-command-container-kubectl-exec.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caesarxuchao
+- mikedanese
+title: Running Commands in a Container using kubectl exec
+---
+
+{% capture overview %}
+
+This page shows how to use `kubectl exec` to run commands in a Container.
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## Using kubectl exec to view the environment variables of a Container
+
+Kubernetes exposes
+[services](/docs/user-guide/services/#environment-variables) through environment variables.
+It is convenient to check these environment variables using `kubectl exec`.
+
+In this exercise, you create a Service and a Deployment. Here is the
+configuration file for the Service:
+
+{% include code.html language="yaml" file="redis-service.yaml" ghlink="/docs/tasks/debug-application-cluster/redis-service.yaml" %}
+
+In the configuration file, you can see that the Service provides access to
```

Comment:
This frontloads a bunch of complexity.  Lets just start with creating a simple Deployment without a Service, and show the user how to run "exec" to get a bash shell with -t -i, and then run top or something.